src/templates/next/components/KeyStatistics/KeyStatistics.tsx

```diff
@@ -1,7 +1,26 @@
 import { type KeyStatisticsProps } from "~/common"
 import { ComponentContent } from "../shared/customCssClass"
 
+const ITEM_WIDTHS: Record<
+  KeyStatisticsProps["variant"],
+  Record<number, string>
+> = {
+  side: {
+    1: "basis-full",
+    2: "basis-[calc((100%-2.5rem)/2)]",
+    3: "basis-[calc((100%-5rem)/3)]",
+  },
+  top: {
+    1: "basis-full",
+    2: "basis-[calc((100%-2.5rem)/2)]",
```

Comment:
do we need to do these calc ourselves or can consider using presets?

Reply:
So far this is something that is quite specific for key statistics component, so opted to not use presets for this.